src/Microsoft.VisualStudio.Editors/PropPages/SeparatorGroupBox.vb

```diff
@@ -0,0 +1,40 @@
+﻿' Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+Imports System.Drawing
+Imports System.Windows.Forms
+
+Namespace Microsoft.VisualStudio.Editors.PropertyPages
+
+    ''' <summary>
+    ''' This class implements a <see cref="GroupBox"/> with custom appearance.
+    ''' </summary>
+    ''' <remarks>
+    ''' The control is rendered as a label followed by a horizontal line, commonly used as a separator.
+    ''' </remarks>
+    Friend NotInheritable Class SeparatorGroupBox
+        Inherits GroupBox
+
+        Private Const LabelToLineDistance As Integer = 8
```

Comment:
>Private Const LabelToLineDistance As Integer = 8 [](http://example.com/codeflow?start=8&length=48)

Does this vary with resolution?

Reply:
My understanding is that everything in Winforms is potentially scaled depending on resolution/DPI. So these are "base pixels", not necessarily equal to what's rendered to screen.

Here's how `GroupBox` defines its default padding, for example. Also a constant.
https://github.com/dotnet/winforms/blob/977ad72491afb38ce4f80c754b4f4257d37ed8ca/src/System.Windows.Forms/src/System/Windows/Forms/GroupBox.cs#L133